packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2058,7 +2062,7 @@ class _InputDecoratorState extends State<InputDecorator> with TickerProviderStat
   }
 
   Color _getDefaultIconColor(ThemeData themeData) {
-    if (!decoration.enabled)
+    if (!decoration.enabled && !isFocused)
```

Comment:
I don't fully understand the changes in this file. If the widget is focused, there's no longer a way to tell from its visuals whether it's enabled or not? Is this in line with material spec?

/cc @HansMuller

Reply:
No, there's definitely no Material spec for this.

That's something I think I'll be asking the customer to push for, since this is a use case that probably needs some design work.